virtual-desktop/src/app/plugin-manager/plugin-factory/iframe/iframe-plugin.component.ts

```diff
@@ -32,7 +33,8 @@ export class IFramePluginComponent {
     @Inject(Angular2InjectionTokens.VIEWPORT_EVENTS) private viewportEvents: Angular2PluginViewportEvents,
     @Inject(Angular2InjectionTokens.PLUGIN_DEFINITION) private pluginDefintion: ZLUX.ContainerPluginDefinition,
     @Inject(Angular2InjectionTokens.LAUNCH_METADATA) private launchMetadata: any,
-    @Inject(Angular2InjectionTokens.SESSION_EVENTS) private sessionEvents: Angular2PluginSessionEvents
+    @Inject(Angular2InjectionTokens.SESSION_EVENTS) private sessionEvents: Angular2PluginSessionEvents,
+    @Inject(Angular2InjectionTokens.THEME_EVENTS) private themeEvents: Angular2PluginThemeEvents
```

Comment:
@Optional() is needed because you did not implement this for simplewindowmanager